backstage/README.md

```diff
@@ -0,0 +1,59 @@
+# Agent Check: backstage
+
+## Overview
+
+[Backstage][1] is an open platform for building Developer Portals. This integration enables you to embed DataDog graphs and dashboards into your Backstage instance.
+
+## Setup
+
+### Installation
+
+1. Install the DataDog plugin into Backstage:
+
+```shell
+cd packages/app
+yarn add @roadiehq/backstage-plugin-datadog
+```
+
+2. Add the DataDog plugin widget to your Backstage Overview tab ([detailed instructions][2]).
+3. Get the [public URL][3] for your DataDog Dashboard. 
+4. Add the Dashboard URL to the plugin's metadata:
+
+```yaml
+apiVersion: backstage.io/v1alpha1
+kind: Component
+metadata:
+  name: sample-service
+  description: |
+    A sample service
+  annotations:
+    datadoghq.com/dashboard-url: <<DATADOGURL>>
+```
+
+### Screenshot
+
+![Screenshot of Backstage's Overview featuring a DataDog graph](https://raw.githubusercontent.com/RoadieHQ/roadie-backstage-plugins/main/plugins/frontend/backstage-plugin-datadog/docs/datadog-widget.png)
```

Comment:
Can you move this image into our repo so it can be displayed correctly? Thanks!

Reply:
I'll remove the image as suggested by @buraizu, thanks!